renormalizer/cv/tests/test_abs.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+
+from renormalizer.cv.zerot import SpectraZtCV
+from renormalizer.cv.finitet import SpectraFtCV
+import numpy as np
+from renormalizer.tests.parameter import mol_list
+import os
+from renormalizer.cv.tests import cur_dir
+from renormalizer.utils import Quantity, CompressConfig
+import pytest
+
+
+def test_zt_abs(switch_to_64backend):
```

Comment:
`switch_to_64backend` is no longer available on master and tests are all run in 64-bit backend.